clientd3d/statgame.c

```diff
@@ -15,6 +15,15 @@
 /* Current state of game--selecting object, picking character, etc. */
 static int game_state = GAME_NONE;
 
+extern int main_viewport_height;
+extern int main_viewport_width;
+
+bool text_area_resize_zone = false;
```

Comment:
Assuming these variables aren't accessed from outside this file, declare them static so they're not visible.